src/pages/Calc.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Calculator from '../components/Calculator';
+import './Calc.css';
+
+const Calc = () => (
+  <div className="calc-holder">
+    <h2 className="sub-title">E=MC2</h2>
+    <Calculator />
+  </div>
```

Comment:
## Optional
- [ ] I did not get the meaning of this `h2` I suggest that you add a meaningful one to make a user XP so easy